c_tests/tests/intrinsics.c

```diff
@@ -0,0 +1,46 @@
+// Compiler:
+//   env-var: YKD_PRINT_JITSTATE=1
+// Run-time:
+//   env-var: YKD_PRINT_IR=jit-pre-opt
+//   stderr:
```

Comment:
Would I be an idiot if I said I couldn't see anything about `memcpy` in the test?

Reply:
You wouldn't. But since `memcpy` is being inlined and we don't have negative matches yet to check that it doesn't appear, there's not much we can test here, unfortunately.

Reply:
Aha, makes sense.

Reply:
I was about to say: why don't we check for the presence of inlining metadata, but that wouldn't be portable... so, I agree, we can't do much more.